frame/vamm/src/mock.rs

```diff
@@ -0,0 +1,125 @@
+// use super::currency_factory::MockCurrencyId;
```

Comment:
why this code is commented out?

Reply:
I commented out just because I wasn't using any of these yet (but I will in the next days).

Is it preferable to just let all the imports there uncommented, regardless if they are being used or not?